ITensorGPU/src/tensor/cudense.jl

```diff
@@ -567,7 +567,7 @@ function Base.permute!(B::CuDense, Bis::IndexSet, A::CuDense, Ais::IndexSet)
     reshapeBdata,
     Vector{Char}(ctbinds),
   )
-  return Tensor(Bis, Dense(vec(reshapeBdata)))
+  return Tensor(Tuple(Bis), Dense(reshapeBdata))
```

Comment:
As we discussed, I think we should move away from versions of `Tensor` with the input ordering `Tensor(inds, storage)` and only allow `Tensor(storage, inds)`. I don't think we need to do that deprecation in this PR, but we should try to follow that convention.